config/config.go

```diff
@@ -455,8 +456,9 @@ func (c *Config) ProcessDataTables() (err error) {
 			if c.DataTable[i].RollupAutoTable != "" {
 				table = c.DataTable[i].RollupAutoTable
 			}
+			interval := c.DataTable[i].RollupAutoInterval
 
-			c.DataTable[i].Rollup, err = rollup.NewAuto(c.ClickHouse.URL, table, time.Minute, rdp, rdf)
+			c.DataTable[i].Rollup, err = rollup.NewAuto(c.ClickHouse.URL, table, interval, rdp, rdf)
```

Comment:
I'd like to keep it as it is, 1 minute. It doesn't harm, but everybody will be surprised if eventually, the auto-rollup update will require service restart